.cirrus.yml

```diff
@@ -54,6 +69,39 @@ task:
     - export TZ=utc
     - bash .ci-scripts/x86-64-apple-darwin-nightly.bash ${CLOUDSMITH_API_KEY}
 
+task:
+  only_if: $CIRRUS_API_CREATED == "true"
+
+  windows_container:
+    dockerfile: .ci-dockerfiles/windowsservercore/Dockerfile
+    os_version: 2019
+    cpu: 8
+    memory: 24
+
+  name: "nightly: x86-64-pc-windows-msvc"
+
+  environment:
+    CLOUDSMITH_API_KEY: ENCRYPTED[!2cb1e71c189cabf043ac3a9030b3c7708f9c4c983c86d07372ae58ad246a07c54e40810d038d31c3cf3ed8888350caca!]
+
+  libs_cache:
+    folder: build/libs
+    fingerprint_script:
+      - ps: (Get-FileHash -Path lib\CMakeLists.txt).Hash
+    populate_script:
+      - ps: .\make.ps1 -Command libs -Generator "Visual Studio 16 2019"
+
+  config_script:
```

Comment:
could we roll all of this up into one script that is in .ci-scripts ala macOS and Linux scripts? Or is that not something that is desirable with windows?